src/hooks/room/useRoomCall.ts

```diff
@@ -258,6 +265,22 @@ export const useRoomCall = (
         });
     }, [isViewingCall, room.roomId]);
 
+    const generateCallLink = useCallback(() => {
+        const url = new URL(SdkConfig.get("element_call").url ?? DEFAULTS.element_call.url!);
+        url.pathname = "/room/";
+        // Set params for the sharable url
+        url.searchParams.set("roomId", room.roomId);
+        url.searchParams.set("perParticipantE2EE", "true");
+        url.searchParams.set("viaServers", "matrix.org");
```

Comment:
uh, really?

Reply:
What is the question rellated to?
The viaServers wording? That it is in there? That we add this to the url?

Reply:
Surely it shouldn't be hardcoded to matrix.org?

Reply:
The whole MSC is should really be a draft.

Reply:
What would be the most recommended param for setting the viaServers?
The current client HS? Or is there sth like a getRecommendedHomeserver on the Room obj?

Reply:
[The spec tells you how to calculate vias](https://spec.matrix.org/v1.9/appendices/#routing)

Reply:
Is there any helper/convinience method in the js sdk?

Reply:
Not in the JS SDK. React SDK has `calculateRoomVia`